packages/sdk/src/modules/loggerModule/utils.ts

```diff
@@ -0,0 +1,13 @@
+import type { LoggerModuleConfig } from "./types";
+
+export const isEmptyObject = (obj = {}) => {
+  return Object.keys(obj).length === 0;
+};
+
+export const isInvalidConfig = (config?: LoggerModuleConfig) => {
```

Comment:
Please, add a docblock that will shortly explain what is "invalid" for us, because it is not invalid in common sense but it does not apply to custom logger.

Reply:
Good idea, we could forget about what is invalid in some time. 
It's added now.